rasa/utils/__init__.py

```diff
@@ -0,0 +1,97 @@
+import io
```

Comment:
Can we create a module `rasa.utils.io` (or maybe you can come up with a better name) where we put these? We are currently trying to get rid of our code in the `__init__.py` files (the current code base does not reflect that yet 😄 )